app/models/katello/puppet_module.rb

```diff
@@ -26,5 +56,32 @@ def self.parse_metadata(filepath)
     ensure
       tar_extract.close if tar_extract
     end
+
+    def self.with_identifiers(ids)
+      self.where(:id => ids)
+    end
```

Comment:
I think the method didn't exist when I first wrote this, I'll remove it in favor of the PulpContentUnit method.